YvesIraguha/send_it_challenge
# Ch-setup-database-162110606

#### What does this PR do?
This PR connects the app to database. 
#### Description of Task to be completed?
* Install PostgreSQL database and connect to it. 
* Perform CRUD operations. 
* Create tests for CRUD operations. 
#### How should this be manually tested?
* Install Postgresql 
* Create connections to the database 
* Run `npm test` to see the ouput of the test
#### Any background context you want to provide?
This branch was created in order to remove data from memory and store them into a database
#### What are the relevant pivotal tracker stories?
#162110606
